### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by SpeedyDevil**
Fixed Meditation skill check
Fixed eastToWest calc of magic fields

### CODE REVIEW

File: Projects/UOContent/Skills/Meditation.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public static TimeSpan OnUse(Mobile m)
             }
 
             var skillVal = m.Skills.Meditation.Value;
-            var chance = (50.0 + (skillVal - (m.ManaMax - m.Mana)) * 2) / 100;
+            var chance = (50.0 + ((skillVal - (m.ManaMax - m.Mana)) * 2)) / 100;
```
Comment (kamronbatman) [inline Projects/UOContent/Skills/Meditation.cs:74]: Double parenthesis, let's remove them.

File: Projects/UOContent/Spells/Base/SpellHelper.cs [hunk 2]
```diff
@@ -220,7 +220,7 @@ public static bool CheckCombat(Mobile m)
                 }
             }
 
-            if (Core.Expansion == Expansion.AOS)
+            if (Core.AOS)
```
Comment (kamronbatman) [inline Projects/UOContent/Spells/Base/SpellHelper.cs:223]: I think this was supposed to specifically be for only AOS and not later?
Let's double check if possible what the different eras should be.

File: Projects/UOContent/Spells/Base/SpellHelper.cs [hunk 3]
```diff
@@ -257,6 +257,17 @@ public static bool AdjustField(ref Point3D p, Map map, int height, bool mobsBloc
             return false;
         }
 
+
+        public static bool GetEastToWest(IPoint3D from,IPoint3D target)
```
Comment (kamronbatman) [inline Projects/UOContent/Spells/Base/SpellHelper.cs:261]: We should already have this somewhere I thought. I'll double check when I get back.

File: Projects/UOContent/Spells/Base/SpellHelper.cs (hunk 2 above)
Comment (SpeedyDevil) [reply Projects/UOContent/Spells/Base/SpellHelper.cs:223]: ServUO doesnt check it for expansion at all.

File: Projects/UOContent/Spells/Base/SpellHelper.cs (hunk 3 above)
Comment (SpeedyDevil) [reply Projects/UOContent/Spells/Base/SpellHelper.cs:261]: I didnt find it anywhere. In spells there are separately the condition (and not working :) )

File: Projects/UOContent/Skills/Meditation.cs (hunk 1 above)
Comment (kamronbatman) [inline Projects/UOContent/Skills/Meditation.cs:74]: I checked and I don't think this does anything.
Screen Shot 2021-09-18 at 8 28 11 PM

Note the gray parenthesis

Comment (SpeedyDevil) [reply Projects/UOContent/Skills/Meditation.cs:74]: hmm ,I wonder why I edited it. ServUO have it with parenthesis but it doesnt change anything (only reading is better with it).

**Review by kamronbatman — APPROVED** [review]

**Pull request merged**